crates/iota-config/data/fullnode-template-testnet.yaml

```diff
@@ -0,0 +1,32 @@
+# Update this value to the location you want IOTA to store its database
+db-path: /opt/iota/db
+
+# For ipv4, update this to "/ipv4/X.X.X.X/tcp/8080/http"
+network-address: /dns/localhost/tcp/8080/http
+metrics-address: "0.0.0.0:9184"
+# this address is also used for web socket connections
+json-rpc-address: "0.0.0.0:9000"
+enable-event-processing: true
+
+genesis:
+  # Update this to the location of where the genesis file is stored
+  genesis-file-location: /opt/iota/config/genesis.blob
+
+# Update this to the location of where the migration file is stored
+migration-tx-data-path: /opt/iota/config/migration.blob
+
+authority-store-pruning-config:
+  num-latest-epoch-dbs-to-retain: 3
+  epoch-db-pruning-period-secs: 3600
+  num-epochs-to-retain: 1
+  max-checkpoints-in-batch: 10
+  max-transactions-in-batch: 1000
+  pruning-run-delay-seconds: 60
+
+# Peering configuration for testnet
+p2p-config:
+  seed-peers:
+    - address: /dns/access-0.r.testnet.iota.cafe/udp/8084
+      peer-id: 46064108d0b689ed89d1f44153e532bb101ce8f8ca3a3d01ab991d4dea122cfc
+    - address: /dns/access-1.r.testnet.iota.cafe/udp/8084
+      peer-id: 8ffd25fa4e86c30c3f8da7092695e8a103462d7a213b815d77d6da7f0a2a52f5
```

Comment:
This file is missing the state-sync-archive for testnet

Reply:
Fixed in https://github.com/iotaledger/iota/pull/5161/commits/f0ff0909284521d1b20cf1429dca99888988612d .